### PR TITLE
Adds tag basic

### PR DESCRIPTION
**Pull request by srambach**
Summary of changes: Add the tag basic element

### CODE REVIEW

File: source/scss/_pf-basic-tags.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
+// Label
+// --------------------------------------------------
+
+.tag {
```
Comment (andresgalante) [inline source/scss/_pf-basic-tags.scss]: All of these should be set on variables. ping me when you are online and I'll show you how, its super easy 😄

File: source/_patterns/basics/10-tags/tag.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+NOTE: Bootstrap 4 renames Labels to be Tags in order disambiguate with the <label> tag. See [http://getbootstrap.com/components/#labels](http://getbootstrap.com/components/#labels) for complete labels documentation.
+
+Patternfly design specs are at [http://www.patternfly.org/pattern-library/widgets/#labels](http://www.patternfly.org/pattern-library/widgets/#labels)
+
+Must add a modifier class to add color to the label.
+
+NOTE: Tags contained within `H1`..`H6` are styled slightly differently.
+
+## Accessibility
```
Comment (andresgalante) [inline source/_patterns/basics/10-tags/tag.md]: We can remove accessibility and usage info from bs components, its already on their docs.

File: source/_patterns/basics/10-tags/tag.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: Tag
+---
+## Overview
+
+This is a Bootstrap component, [see complete documentation here](http://v4-alpha.getbootstrap.com/components/tag/)
+
+NOTE: Bootstrap 4 renames Labels to be Tags in order disambiguate with the <label> tag. See [http://getbootstrap.com/components/#labels](http://getbootstrap.com/components/#labels) for complete labels documentation.
```
Comment (andresgalante) [inline source/_patterns/basics/10-tags/tag.md]: I like the `NOTE`, lets add it to the template and make it `**Note:**` so its bold

**Comment by srambach** [comment]
Updated again to fix what I messed up in git. @cshinn @andresgalante please review :-)

**Comment by srambach** [comment]
@andresgalante one more time with feeling!

**Comment by andresgalante** [comment]
👏  great job sarah!

**Comment by cshinn** [comment]
Looks good!

**Comment by jgiardino** [comment]
LGTM!

**Pull request merged**